README.md

```diff
@@ -1 +1,11 @@
-# Sprouts
\ No newline at end of file
+# The Sprouts Game
+Invented in 1967, Sprouts is a two player game that starts with any number of dots randomly placed. Taking turns, players will connect two dots with a line (this includes starting and ending a line at the same dot) and adding a new dot somewhere along this line after which their turn ends. The game ends when no more connections can be made and the player to draw the last valid line is the winner. They are constrained by the following rules:
+
+1. The line may take on any shape, but not get too close to itself, or cross existing lines.
+2. The new dot may not be placed on an existing dot. The new dot will split an existing line into two shorter lines.
+3. No dot can have more than three lines attached to it, so new dots placed are considered to have two lines attached to it already.
+
+The idea is to make it impossible for the other player to win. 
+
+Rules obtained : https://nrich.maths.org/2413
+
```

Comment:
It would be nice to add pictures here, like there are in the link you provided, to give the reader/user a better idea of how the game works and how it will look like